src/Components/News/Detail/NewsDetailLayout.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect,useState } from 'react';
+import { useParams } from 'react-router';
+import NewsTitle from './NewsTittle';
+import { Box } from '@material-ui/core';
+import { getNewById } from '../../../Services/getNewById';
+
+const NewsDetailLayout = ({prop}) => {
+
+    const [newData,setNewData] = useState({})
+
+    const {id} = useParams()
+
+    useEffect(()=>{
+        const loadNewData = async () => {
```

Comment:
the useffect should not have functions declarations. please move this outside the useEffect

Reply:
Done! :)